tools/ci_build/github/azure-pipelines/orttraining-pai-ci-pipeline.yml

```diff
@@ -65,6 +65,7 @@ jobs:
   - script: |-
       cd ./build/RelWithDebInfo
       export PYTHONPATH=$PWD
+      export ORTMODULE_ONNX_OPSET_VERSION=12
```

Comment:
Are we not able to update the torch versin for this pipeline?

Reply:
I guess we could. don't know which value to choose for this parameter: https://github.com/microsoft/onnxruntime/blob/master/tools/ci_build/github/pai/rocm-ci-pipeline-env.Dockerfile#L1.